TestApp/ForwardOnlyStream.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.IO;
+
+namespace TestApp
+{
+    class ForwardOnlyStream : Stream
```

Comment:
Any reason to include this file? It doesn't seem to be used at all. Out of curiosity why limit a stream like this?

Reply:
That's there so I can force use of the ForwardOnlyPageReader in ContainerReader.  I don't use it in the committed code, but do use it for testing.